plugin-server/src/worker/ingestion/internal-handlers/handlers/surveys/surveySent.ts

```diff
@@ -0,0 +1,18 @@
+import { PluginEvent } from '@posthog/plugin-scaffold'
+
+import { InternalEventHandler, InternalEventHandlerContext } from '../../registry'
+
+// check if the survey needs to be stopped when a response is submitted
+export const surveySentHandler: InternalEventHandler = {
+    name: 'surveys sent',
+    events: ['survey sent'],
+
+    async handle(event: PluginEvent, context: InternalEventHandlerContext): Promise<void> {
+        const surveyId = event.properties?.['$survey_id']
+        if (surveyId && context.celery) {
+            await context.celery.applyAsync('posthog.tasks.tasks.stop_surveys_reached_target', [], {
```

Comment:
We can't introduce Celery to ingestion:

- This is yet another dependency, so it means lower reliability and more potential incidents
- We already have a pattern for this - we build dowstream services based on ingestion output topics, which then do the heavy lifting without slowing down the ingestion path.
- I've seen several issues with Celery, I don't know how well it is maintained (and by what team), and it feels like a liability for ingestion